### DISCUSSION TITLE
Make using throwaway UsePrivacyMode boxes easier to work?

### DISCUSSION THREAD

**Discussion by stdedos** [sandboxie-plus/Sandboxie#4253] [Q&A Feedback]
When using `/box:__ask__` to launch an app, would you consider adding the parent directory of the executable into the box config?

e.g. launching `F:\Some\Portable\Path\app.exe` would add `OpenFilePath=F:\Some\Portable\Path\` to the configuration

**Comment by wilders-soccerfan** [comment]
For UsePrivacyMode boxes, perhaps `NormalFilePath=...` might be a more relevant fit, if going this route.

**Reply by stdedos** [reply]
Well ... it depends 😅 

e.g. for a portable app, it writing a `settings.ini` to its own folder doesn't bother me.

Ofc I can see the arguments against (e.g. an app self-auto-updates, and you **definitely** don't want that)
